mkdocs/docs/experiments/faq/install.md

```diff
@@ -0,0 +1,79 @@
+---
+hide:
+  - toc
+---
+# Install
+
+## Table of Contents
+
+<style> 
+   /* Hide table of contents */
+      .md-sidebar--secondary {
+        display: none;
+      }
+  @media only screen and (min-width: 60em) {
+  .md-content {
+    margin: 0;
+  }
+  }
+</style>
```

Comment:
will this work with md file?

Reply:
yes, it will work. But we are overriding this so removing this css